web/vtadmin/src/components/routes/Workflows.tsx

```diff
@@ -112,13 +112,53 @@ export const Workflows = () => {
+                                    var numThrottled = 0;
+                                    var throttledApp = '';
+                                    var throttledFrom: vttime.ITime | null | undefined;
                                     const streamCount = row.streams[streamState].length;
+                                    var streamDescription: string;
+                                    switch (streamState) {
+                                        case 'Error':
+                                            streamDescription = 'failed';
+                                            break;
+                                        case 'Running':
+                                            const running = row.streams['Running'];
+                                            if (running !== undefined && running !== null) {
+                                                for (const stream of running) {
+                                                    if (
+                                                        stream?.throttler_status?.component_throttled !== null &&
+                                                        stream?.throttler_status?.component_throttled !== undefined
+                                                    ) {
+                                                        numThrottled++;
+                                                        throttledApp = stream?.throttler_status?.component_throttled;
+                                                        throttledFrom = stream?.throttler_status?.time_throttled;
+                                                    }
+                                                }
+                                            }
+                                            if (numThrottled > 0) {
+                                                streamDescription = '';
+                                                streamState = 'Throttled';
+                                            } else {
+                                                streamDescription = streamState;
+                                            }
+                                            break;
+                                        default:
+                                            streamDescription = streamState.toLocaleLowerCase();
+                                    }
                                     const tooltip = [
                                         streamCount,
-                                        streamState === 'Error' ? 'failed' : streamState.toLocaleLowerCase(),
+                                        streamDescription,
                                         streamCount === 1 ? 'stream' : 'streams',
+                                        numThrottled > 0
+                                            ? '(' +
+                                              numThrottled +
+                                              ' throttled by ' +
```

Comment:
`component throttled` or `application throttled` is the more accurate/better terminology rather than `throttled by` IMO. `throttled in` could also be used if you prefer.

Reply:
Changed to " in ".